chapters/VK_KHR_swapchain/wsi.txt

```diff
@@ -1166,6 +1166,18 @@ the presentation engine.
 However, the scope of this set of queue operations does not include the
 actual processing of the image by the presentation engine.
 
+.Note
+[NOTE]
+====
+The origin of the native orientation of the surface coordinate system is not
+specified in the Vulkan specification; it depends on the platform.
+For most platforms the origin is by default upper-left, meaning the pixel
+of the presented slink:VkImage at coordinates [eq]#(0,0)# would appear at
+the upper left pixel of the platform surface (assuming
+ename:VK_SURFACE_TRANSFORM_IDENTITY_BIT_KHR, and the display standing the
+right way up).
+====
+
```

Comment:
The note feels a little arbitrary sitting here among queue operation scope language and whatnot, but there doesn't seem to be any great place for this note to live, so I guess this is as good as any.

Reply:
The note is below `vkQueuePresentKHR`, where it feels most useful to me.

Yea, I have been strugling to find the best place. There still kinda feels like there is some language missing describing the `vkQueuePresent` behavior in general, which would be my first choice to put the Note there.

The previous sentence at least talks about the swapchain images, which felt like the least non-relevant place to put the Note in.